lib/mobius/actions/message.ex

```diff
@@ -0,0 +1,52 @@
+defmodule Mobius.Actions.Message do
+  @moduledoc """
+  Actions related to Discord messages such as sending, editing, and deleting messages
+  """
+
+  alias Mobius.Models.Message
+  alias Mobius.Rest
+  alias Mobius.Services.Bot
+
+  @doc """
+  Send a message in a channel
+
+  The params can be:
+    - `content`: The message's content (maximum 2000 chars)
+    - `nonce`: random string or integer to compare with received messages and check its reception
+    - `tts`: true if this message should be read with tts
+    - `embed`: an embed (with type `rich`) with some restrictions
+    - `allowed_mentions`: the mentions allowed in this message (see docs)
+    - `message_reference`: identifies the message being replied to (see docs)
```

Comment:
```suggestion
    - `embed`: an embed with type `rich` and some restrictions (see Discord docs)
    - `allowed_mentions`: the mentions allowed in this message (see Discord docs)
    - `message_reference`: identifies the message being replied to (see Discord docs)
```

I know we're in the context of a Discord bot framework but "see docs" inside docs feels a little circular 😛